File: C/1011.c
#include <stdio.h>
int main(void)
{
    int R;
    double pi = 3.14159, VOLUME;
    scanf("%d", &R);
    VOLUME = (4 / 3.0) * pi * R * R * R;
    printf("VOLUME = %.3lf\n", VOLUME);
    return 0;
}

File: Javascript/1011.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

const raio = parseFloat(lines[0])
const pi = 3.14159
const volume = (4/3) * pi * Math.pow(raio, 3)

console.log(`VOLUME = ${volume.toFixed(3)}`)
